File: Algorithms/C++/0412/0412.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> v(n);
        for (int i = 1; i <= n; i++) {
            if (i % 3 == 0) {
                v[i-1] = i % 5 ? "Fizz" : "FizzBuzz";
            } else if (i % 5 == 0) {
                v[i-1] = "Buzz";
            } else {
                stringstream ss;
                ss << i;
                ss >> v[i-1];
            }
        }
        return v;
    }
};

File: Algorithms/Python/0412/0412.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        myList = []
        for i in range(1, n + 1):
            if i % 3 == 0:
                myList.append("FizzBuzz" if i % 5 == 0 else "Fizz")
            elif i % 5 == 0:
                myList.append("Buzz")
            else:
                myList.append(str(i))
        return myList
